app/presenters/publishing_api/links_presenter.rb

```diff
@@ -65,6 +65,8 @@ def world_location_ids
     end
 
     def worldwide_organisation_ids
+      return (item.try(:editionable_worldwide_organisations) || []).map(&:content_id) if item.try(:editionable_worldwide_organisations)&.any?
```

Comment:
So, this is based on the assumption that models will only have editionable worldwide organisations after the migration.

We can alternatively use the feature flag here, but if we do so this won't be fully-functional when using the cookies strategy (maybe not a problem)

Reply:
I'm now thinking this a is bad assumption and we should just use the feature flag

Reply:
I think this is a good assumption, assuming that migration works exactly as planned feels wobbly to me

Reply:
We decided what I've done is fine here. I've highlighted it on the migration card so we can address it when we have more info.